src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -1,11 +1,22 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Excavator
- * Do not remove no-args constructor
- */
 public class Excavator extends Machine {
+    private double bucketCapacity;
+
+    public Excavator(String name, double bucketCapacity, String color) {
```

Comment:
same

Reply:
Because It is in Machine class and Excavator extends Machine. I added only some type-specific fields to Excavator classes.